src/site/content/en/blog/largest-contentful-paint/index.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Largest Contentful Paint
+subhead: An important, new user-centric performance metric
+authors:
+  - philipwalton
+date: 2019-08-08
+hero: hero.jpg
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+hero_position: top
+alt: Artist painting tools
+description: |
+  This post introduces the Largest Contentful Paint API, how to use it,
+  and offers advice on how to optimize your pages for this metric
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+  - metrics
+---
+
+Historically, it's been a challenge for web developers to measure how quickly
+the main content of a web page loads and is visible to users.
+
+Older metrics like
+[load](https://developer.mozilla.org/en-US/docs/Web/Events/load) or
+[DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/Events/DOMContentLoaded)
+are not good because they don't necessarily correspond to what the user sees on
+their screen. And newer, user-centric performance metrics like [First Paint
+(FP) and First Contentful Paint
+(FCP)](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics#first_paint_and_first_contentful_paint)
+only capture the very beginning of the loading experience. If a page shows a
+splash screen or displays a loading indicator, this moment is not very relevant
+to the user.
+
+In the past we've recommended performance metrics like [First Meaningful Paint
+(FMP)](https://web.dev/first-meaningful-paint/) and [Speed Index
+(SI)](https://web.dev/speed-index/) (both available in Lighthouse) to help
+capture more of the loading experience after the initial paint, but these
+metrics are complex, hard to explain, and often wrong&mdash;meaning they still do
+not identify when the main content of the page has loaded.
+
+Sometimes simpler is better, and the latest research done in the [Web
+Performance Working Group](https://www.w3.org/webperf/) suggests that a more
+accurate way to measure when the main content of a page is loaded is to look at
+when the largest element was rendered.
+
+## Largest Contentful Paint defined
+
+The [Largest Contentful Paint
+(LCP)](https://wicg.github.io/largest-contentful-paint/) API, available in
+Chrome 77 beta, reports the render time of the largest content element visible
+in the viewport.
+
+### What elements are considered?
+
+As currently specified, the types of elements considered for Largest Contentful
+Paint are:
+
+* `<img>` elements
+* `<image>` elements inside an `<svg>` element
+* Any element with a background image that is fetched (as opposed to a gradient)
+* Block-level elements containing text nodes or other inline-level text elements
+  children.
+
+Note, restricting the elements considered to this limited set was intentional
```

Comment:
Delete 'considered'. It doesn't contribute meaning.

Reply:
Done.